docs/fundamentals/snippets/testing/AspireApp1/AspireApp1.ApiService/Program.cs

```diff
@@ -0,0 +1,39 @@
+var builder = WebApplication.CreateBuilder(args);
+
+// Add service defaults & Aspire components.
+builder.AddServiceDefaults();
+
+// Add services to the container.
+builder.Services.AddProblemDetails();
+
+var app = builder.Build();
+
+// Configure the HTTP request pipeline.
+app.UseExceptionHandler();
+
+var summaries = new[]
+{
+    "Freezing", "Bracing", "Chilly", "Cool", "Mild", "Warm", "Balmy", "Hot", "Sweltering", "Scorching"
+};
```

Comment:
We give you these awesome features like collection expressions:

```suggestion
string [] summaries = [ "Freezing", "Bracing", "Chilly", "Cool", "Mild", 
                                     "Warm", "Balmy", "Hot", "Sweltering", "Scorching"];
```

Reply:
Yeah, I agree. This was code from their templates though, and until they change the template - we'd best not deviate from those.